src/hooks/useThumbnailDimensions.ts

```diff
@@ -0,0 +1,18 @@
+import {useMemo} from 'react';
+import CONST from '@src/CONST';
+import useWindowDimensions from './useWindowDimensions';
+
+export default function useThumbnailDimensions(width: number, height: number) {
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const fixedDimenstion = isSmallScreenWidth ? CONST.THUMBNAIL_IMAGE.SMALL_SCREEN.SIZE : CONST.THUMBNAIL_IMAGE.WIDE_SCREEN.SIZE;
+
+    const thumbnailDimensionsStyles = useMemo(() => {
+        const aspectRatio = width / height || 1;
```

Comment:
If we care about `width === 0`, then what if the height is `0`? 
![Screenshot 2024-01-23 at 23 03 33](https://github.com/Expensify/App/assets/104823336/22fdd5df-6127-439d-a64a-b35135703007)